docs/tutorials/custom_builder_go.md

```diff
@@ -1,87 +1,105 @@
+- [stacks](https://buildpacks.io/docs/concepts/components/stack/)
+- [builders](https://buildpacks.io/docs/concepts/components/builder/)
+- [buildpacks](https://buildpacks.io/docs/concepts/components/buildpack/).
 
+You should also have [`pack`](https://buildpacks.io/docs/tools/pack/) installed,
+along with `docker` and `skopeo` (to manage Open Container Initiative (OCI) artifacts).
 
 ## Create the stack
 
-Unfortunately we were not able to reuse the [Paketo Full Stack](https://github.com/paketo-buildpacks/bionic-full-stack) because of a missing dependency in the `run` image. To make it simpler we just [forked](https://github.com/epinio/bionic-full-stack) this stack and added `git` to the run packages (see [408e949](https://github.com/epinio/bionic-full-stack/commit/408e949558a437c99858ac7bb99a8b78e78935e8)).
+It's impossible to reuse the [Paketo Full Stack](https://github.com/paketo-buildpacks/bionic-full-stack)
+because of a missing dependency in the `run` image.
+To make it easy there is a [forked](https://github.com/epinio/bionic-full-stack)
+stack and added `git` to the run packages
+(see [408e949](https://github.com/epinio/bionic-full-stack/commit/408e949558a437c99858ac7bb99a8b78e78935e8)).
+
+:::note
+
+We had to fork and reuse the same stack ID so that it's possible to reuse certain Paketo buildpacks.
+They would work only with these stacks
+(see [buildpack.toml#L29](https://github.com/paketo-buildpacks/node-engine/blob/main/buildpack.toml#L29)).
 
-> Note:  
-> We had to fork and/or reuse the same stack ID in order to reuse some Paketo buildpacks, that are going to work only with these stacks (see https://github.com/paketo-buildpacks/node-engine/blob/main/buildpack.toml#L29).
+:::
 
 ### Build the stack and publish the image
 
 To build the stack:
 
-```
+```console
 ./stacks/bionic-full-stack/scripts/create.sh
 ```
 
-This will create two OCI images in the `stacks/bionic-full-stack/build` folder:
+This creates two Open Container Initiative (OCI) images in the `stacks/bionic-full-stack/build` folder:
 
 - build.oci
 - run.oci
 
+From this folder you can un-tar the OCI artifacts:
 
-From this folder we can untar the OCI artifacts
-
-```
+```console
 cd stacks/bionic-full-stack/build
 mkdir build && tar -xf build.oci -C build
 mkdir run   && tar -xf run.oci   -C run
 ```
 
-use `skopeo` to copy the artifacts into our local Docker registry
+Use `skopeo` to copy the artifacts into our local Docker registry:
 
-```
+```console
 skopeo -v copy oci:build docker-daemon:ghcr.io/enrichman/bionic-full-stack-build:0.1.0
 skopeo -v copy oci:run   docker-daemon:ghcr.io/enrichman/bionic-full-stack-run:0.1.0
```

Comment:
Should possibly note early on that `enrichman` is the author of this document, and that the reader should use some other suitable name where it appears. I.e. treat as placeholder for the actual name.

Reply:
Thnaks. I will use '<username>', should be clear enough.